docs/guide/web-simulation.md

```diff
@@ -0,0 +1,259 @@
+- `monitorLogEnabled`: specify if the monitor data have to be stored in a file.
+
+The `/client` request on the simulation server URL will setup a new Webots instance and return the Webots WebSocket URL. The payload have to contain a `init` value with the format `[host, projectPart, worldFilename, user1Id, user1Name, user1Authentication, user2Id, user2Name, customData]`. This data is then used to retrieve the simulation data and setup the Webots project.
+
+The `/monitor` request opens a page showing some information about the current status of the simulation server machines, i.e the number of Webots instances running, the load of the CPU and GPU, the network usage.
+
+The `/load` request returns the current load of the machine computed as the maximum value between all the monitored data (main CPU and GPU load and memory usage, and network usage).
+
+#### Network Settings
+
+Please note that in order to be visible from the outside network, all the port used by the `session_server.py` and `simulation_server.py` scripts should be open (e.g. on simple networks, this can be done by modifying the NAT settings of the router).
+The firewall of the local computer may complain about this operation, in this case, please modify its settings.
+
+#### SSL Encryption
+
+The simulation server works with SSL encryption.
+The application requires the `fullchain.pem` and `privkey.pem` files and their path have to be specified in the `sslKey` and `sslCertificate` values of session and simulation configuration file.
+Note that Webots will look for the file "WEBOTS\_HOME/resources/web/server/ssl/cert.pem", so you may have to rename `fullchain.pem` and copy it in the `ssl` folder or create a soft link.
+
+#### Startup Procedure
+
+The startup procedure is the following:
+
+1. Start all the simulation servers (`simulation_server.py`).
+2. Start the session server (`session_server.py`).
+
+This procedure should be automated in a startup script, so that the servers are restarted after a reboot of the machine.
+
+This folder also contains a `server.sh` utility script to automatically start and stop `session_server.py` and `simulation_server.py` with a given configuration file.
+
+Please make sure that the `WEBOTS_HOME` variable is set before running the simulation and session server scripts.
+
+### Website Host
+
+#### Simulation Data Management
+
+The host where the client website is running should have a `ajax` named folder at the root level containing these scripts:
+* `download-project.php`: a script that returns a zipped archive containing the Webots simulation files to be run. It receives these parameters as POST data:
+  * `user1Id`: id identifying the owner of the project, if an accounted application is used.
+  * `user1Authentication`: user password for the authentication.
+  * `key`: key identifying the sender to prevent brute force attacks.
```

Comment:
This has nothing to do with brute force attacks.

Reply:
Ok. I found this in already existing documentation.
What it is exactly the purpose of the key?

Reply:
I believe it is to authenticate the server. Isn't it the host key?

Reply:
Yes, it is the host key